core/Cargo.toml

```diff
@@ -62,9 +62,9 @@ docs = []
 trust-dns = ["reqwest/trust-dns"]
 
 # Enable rustls for TLS support
-rustls = ["reqwest/rustls-tls-native-roots"]
+rustls = ["reqwest/rustls-tls-native-roots", "redis?/tokio-rustls-comp"]
```

Comment:
I prefer to have services-redis-rustls so users can pick their tls suit on demand

Reply:
Just to be sure i understand what you are asking for, you would like to have to features (`services-redis-rustls` and `services-redis-native-tls`) to select which tls version redis would use ?

But that would allow users to have reqwest with one tls implementation and  redis with another one ? Wouldn't it be an issue ? (sorry if the question is silly, I am just starting to discover opendal)

Also, would it be ok to select `services-redis-rustls` by default ?